manifests/server/etcd.pp

```diff
@@ -65,7 +70,7 @@
       'etcd-server':
         ca_key             => $client_ca_key,
         ca_cert            => $client_ca_cert,
-        addn_names         => delete_undef_values($addn_names), # prevent undef value if ipv6 is turned off
```

Comment:
pls keep delete_undef_values, it would cause trouble for me and every body who has not yet adapted to ipv6